### PR TITLE
snakemake about page

### PR DESCRIPTION
**Pull request by ayushkamat**

### CODE REVIEW

File: latch/types/metadata.py [hunk 1]
```diff
@@ -747,8 +747,28 @@ class SnakemakeMetadata(LatchMetadata):
...
+    about_page_content: Optional[Path] = None
+    """
+    Path to a markdown file containing information about the pipeline - rendered in the About page.
+    """
 
     def validate(self):
+        if self.about_page_content is not None:
+            if not self.about_page_content.exists():
+                click.secho(
+                    f"Path to about page documentation ({self.about_page_content})"
```
Comment (rahuldesai1) [inline latch/types/metadata.py:759]: capital `About` ?

**Review by maximsmol — CHANGES_REQUESTED** [review]
i mean this should really be `about_page_path` but too late to change now

File: latch/types/metadata.py [hunk 2]
```diff
@@ -770,6 +790,12 @@ def __post_init__(self):
         global _snakemake_metadata
         _snakemake_metadata = self
 
+    @property
+    def dict(self):
+        d = super().dict
+        del d["__metadata__"]["about_page_content"]
```
Comment (rahuldesai1) [inline latch/types/metadata.py:796]: why do we need to delete this? maybe add a comment in the code ?

**Review by maximsmol — APPROVED** [review]
sorry, meant to approve lol

**Review by rahuldesai1 — APPROVED** [review]

**Pull request merged**